tests/cypress/integration/wp-module-support/MockApi.cy.js

```diff
@@ -11,6 +11,7 @@ const homepages_mock = require( '../../fixtures/homepages.json' );
 const theme_style_mock = require( '../../fixtures/theme-style.json' );
 const customize_data_mock = require( '../../fixtures/customize-data.json' );
 const homepage_regenerate_mock = require( '../../fixtures/homepage-regenerate.json' );
+const migate_connect_mock = require( '../../fixtures/migrate-connect.json' );
```

Comment:
if you could change it to *migrate_connect_mock

Reply:
Done